Framework/Algorithms/src/ConvertUnits.cpp

```diff
@@ -465,21 +464,18 @@ void ConvertUnits::convertViaTOF(Kernel::Unit_const_sptr fromUnit,
 
       /// @todo Don't yet consider hold-off (delta)
       const double delta = 0.0;
-      std::vector<double> vals(outputWS->x(i).begin(), outputWS->x(i).end());
-      // Convert the input unit to time-of-flight
-      localFromUnit->toTOF(vals, emptyVec, l1, l2, twoTheta, emode, efixed,
-                           delta);
 
+      // TODO toTOF and fromTOF need to be reimplemented outside of kernel
```

Comment:
I am making a note of these things [here](https://github.com/mantidproject/mantid/issues/16118). The these are just for other developers who may encounter this code before we get the chance to change it.